src/database/migrations/z1631541043-update-inflow.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    if (process.env.NODE_ENV === 'test') {
+      return;
+    }
+
+    //add column
+    await queryInterface.addColumn('inflow', 'contractAddress', {
+        type: Sequelize.STRING(44),
+        allowNull: false,
+        defaultValue: '0',
+    });
+
+    await queryInterface.addColumn('inflow', 'asset', {
+      type: Sequelize.STRING(4),
+      defaultValue: 'cUSD',
```

Comment:
I wouldn't do it this way. I think integers here are better, and then we do an enum on code.
Like
`enum Asset {cUSD, cEUR, CELO};`
Reason: I don't see advantages of saving a string on db over using enum on code. But I see advantages on the opposite. It's less likely a bug happens due to some naming issue and it has better performance.
What do you think?

Reply:
sure, is a good ideia. I changed to integer